src/View/Helper/I18nHelper.php

```diff
@@ -91,4 +101,85 @@ public function changeUrlLang($newLang) : string
 
         return $url;
     }
+
+    /**
+     * Translate object field
+     * Return translation (by response object and included data, field and language)
+     *
+     * @param array $object The object to translate
+     * @param array $included The included translations data
+     * @param string $attribute The attribute name
+     * @param string $lang The lang (2 chars string)
+     * @param bool $defaultNull Pass true when you want null as default, on missing translation
+     * @return string|null
+     */
+    public function field(array $object, array $included, string $attribute, string $lang, bool $defaultNull = false) : ?string
+    {
+        $defaultValue = null;
+        if (!$defaultNull) {
+            $defaultValue = Hash::get($object, sprintf('attributes.%s', $attribute));
```

Comment:
if not found in `object.attributes.{name}` we may check in `object.{name}` ? this way we could also use attributes array